src/EnergyPlus/HVACSingleDuctInduc.cc

```diff
@@ -764,7 +795,11 @@ namespace HVACSingleDuctInduc {
 				}
 			} else {
 				CheckZoneSizing( IndUnit( IUNum ).UnitType, IndUnit( IUNum ).Name );
-				MaxTotAirVolFlowDes = max( TermUnitFinalZoneSizing( CurZoneEqNum ).DesCoolVolFlow, TermUnitFinalZoneSizing( CurZoneEqNum ).DesHeatVolFlow );
+				if ( CurTermUnitSizingNum > 0 ) {
```

Comment:
From what I can tell, CurTermUnitSizingNum > 0 when the user enters a name for the DesignSpecification:AirTerminal:Sizing object. They don't have to use that object correct (i.e., blank input)? If that object isn't used then shouldn't sizing be done the same way as before? So what was MaxTotAirVolFlowDes before these changes? It wasn't equal to 0 was it (line 801 below)?